index.js

```diff
@@ -37,6 +37,12 @@ function reproject(srcpath, dstpath) {
   var bandCount = src.bands.count();
   var dataType = src.bands.get(1).dataType;
 
+  try {
```

Comment:
If you're just going to catch the error and throw it, there's no difference from just letting it happen. That is, I don't think you need to try/catch here to have the same effect.